apps/public/components/listings_list.tsx

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import { PropertyTitleImage } from '@dahlia/ui-components/src/cards/property_title_image';
```

Comment:
I don't have a specific problem with this change, but I thought we were going in the direction of more semantically aligned components? This seems like the other direction.

Reply:
in group discussion, decided to go with components that are more generic - e.g. ImageCard instead of PropertyTitleImage - and to in the future keep assessing whether we may want to create more listing-specific components